src/css/corner-layout.scss

```diff
@@ -0,0 +1,100 @@
+// Base styles
+html.easy-popup-open {
+  &:has(.easy-popup-master-corner.open) {
+    .easy-popup-master {
+      opacity: 0;
+      visibility: hidden;
+    }
+  }
+}
+
+.easy-popup-master-corner {
+  transition: opacity 0.4s ease, transform 0.4s ease;
+  transform: translateX(0);
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  max-width: 450px;
+  opacity: 1;
+  visibility: visible;
+  top: unset;
+  left: unset;
+  height: unset;
+  width: unset;
+  display: block;
+
+  &:not(.open) {
+    pointer-events: none;
+  }
+
+  .ep-close-button {
+    position: fixed;
+    transform: translate(0);
+    --ep-close-color: #000;
+  }
+
+  .easy-popup-overflow {
+    overflow: unset;
+  }
+
+  .easy-popup-container {
+    padding: 0;
+
+    .easy-popup-inner {
+      box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+      border-radius: var(--ep-radius);
+      overflow: hidden;
+      max-height: calc(100vh - (var(--ep-corner-offset) * 2));
+      overflow-y: auto;
+    }
+  }
+
+  // Position variants
+  &.top-left {
+    top: var(--ep-corner-offset);
+    left: var(--ep-corner-offset);
+    bottom: unset;
+    right: unset;
+  }
+
+  &.top-right {
+    top: var(--ep-corner-offset);
+    right: var(--ep-corner-offset);
+    bottom: unset;
+  }
+
+  &.bottom-left {
+    bottom: var(--ep-corner-offset);
+    left: var(--ep-corner-offset);
+    right: unset;
+  }
+
+  &.bottom-right {
+    bottom: var(--ep-corner-offset);
+    right: var(--ep-corner-offset);
+  }
+
+  @media only screen and (max-width: 768px) {
```

Comment:
Should this breakpoint be consistent with the mobile layout here https://github.com/viivue/easy-popup/blob/dd83fa3539eda91f598c183298056580bb05be2c/src/css/layout-mobile.scss#L16?